frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -94,18 +103,17 @@ fn submit_candidates_with_self_vote<T: Trait>(c: u32, prefix: &'static str)
 	let candidates = submit_candidates::<T>(c, prefix)?;
 	let stake = default_stake::<T>(BALANCE_FACTOR);
 	let _ = candidates.iter().map(|c|
-		submit_voter::<T>(c.clone(), vec![c.clone()], stake)
+		submit_voter::<T>(c.clone(), vec![c.clone()], stake).map(|_| ())
```

Comment:
This will swallow any errors? Is that okay?

Reply:
Nope, this is a reminiscence of the root PR and should be removed here.